tools_for_todoist/models/event.py

```diff
@@ -30,6 +30,16 @@
 CALENDAR_LAST_COMPLETED = 'last_completed'
 
 
+def is_declined_by_me(raw_event):
```

Comment:
I don't think this should work from a raw_event, but as a `CalendarEvent` method

Reply:
In `_process_sync` you don't have a `CalendarEvent` yet